lib/routes/yilinzazhi/index.ts

```diff
@@ -0,0 +1,58 @@
+import { Data, DataItem, Route } from '@/types';
+import cache from '@/utils/cache';
+import got from '@/utils/got';
+import { load } from 'cheerio';
+
+export const route: Route = {
+    path: '/',
+    categories: ['reading'],
+    example: '/yilinzazhi',
+    radar: [
+        {
+            source: ['www.yilinzazhi.com'],
+            target: '/',
+        },
+    ],
+    name: '文章列表',
+    maintainers: ['g0ngjie'],
+    handler,
+    url: 'www.yilinzazhi.com',
+};
+
+async function handler(): Promise<Data> {
+    const baseUrl = 'https://www.yilinzazhi.com/';
+    const response = await got(baseUrl);
+    const $ = load(response.data);
+    const items: DataItem[] = $('section.content')
+        .find('li')
+        .map((_, target) => {
+            const li = $(target);
+
+            const aTag = li.find('a');
+            const title = aTag.text();
+            const link = baseUrl + aTag.attr('href');
+
+            return {
+                title,
+                link,
+                description: '',
+            };
+        })
+        .toArray();
+
+    await Promise.all(
```

Comment:
Please do return these promises otherwise the `description` will be empty if it's a `cache-hit`.